Jenkinsfile_CNP

```diff
@@ -115,23 +115,23 @@ withPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'build/reports/tests/integration/**/*'
   }
 
-  afterSuccess('functionalTest:preview') {
+  afterAlways('functionalTest:preview') {
```

Comment:
Can you add the following to the two `functionalTest` actions to make the reports easier to find.

``` 
      publishHTML target: [
        allowMissing         : true,
        alwaysLinkToLastBuild: true,
        keepAll              : true,
        reportDir            : "target/cucumber/functional-html-reports",
        reportFiles          : "overview-features.html,overview-failures.html,",
        reportTitles         : "Features,Failures",
        reportName           : "Functional Test Report"
      ]
```